tests/pkl_eval/expected/single_output_specified.txt

```diff
@@ -1 +1 @@
-tests/pkl_eval/pkl_eval_with_out/helloworld.pcf
+tests/pkl_eval/helloworld.pcf
```

Comment:
I'm not sure if removing the intermediary directories is an issue here. I would appreciate feedback on this.

Reply:
It's backward incompatible but I support this change:
- when `outs` is specified, then I strongly prefer not namespacing with a directory (otherwise it's confusing to the user: they specify a path that looks relative to the Bazel package dir and would end up with a different path)
- when `outs` is unspecified with`multiple_outputs = False`, there's already namespacing going on via the file name
- when `outs` is unspecified with `multiple_outputs = True`, namespacing is still happening via a directory.

For context, initially, the rationale for a namespacing directory was to keep every case above consistent (they would all be under a namespaced directory) but taking a step back, the suggested approach has more benefits.